sdk/cognitiveservices/Vision.Face/src/Generated/FaceOperations.cs

```diff
@@ -970,7 +970,8 @@ public async Task<HttpOperationResponse<VerifyResult>> VerifyFaceToFaceWithHttpM
         /// </param>
         /// <param name='returnFaceId'>
         /// A value indicating whether the operation should return faceIds of detected
-        /// faces.
+        /// faces. Detecting with this value set to true requires additional
+        /// permissions.
```

Comment:
"permissions" here is most likely going to be misinterpreted as "admin permission." Perhaps create a fwlink to the gating explanation page and put it here?